src/lib/fileutils.ts

```diff
@@ -298,7 +298,9 @@ export const serviceBuildAndUpdatePipeline = (
                   `echo "Image Name: $IMAGE_NAME"`,
                   `export IMAGE_REPO=${IMAGE_REPO}`,
                   `echo "Image Repository: $IMAGE_REPO"`,
-                  `../fab/fab set --subcomponent $(Build.Repository.Name).$FAB_SAFE_SERVICE_NAME.${SAFE_SOURCE_BRANCH}.chart image.tag=$IMAGE_TAG image.repository=$IMAGE_REPO/$BUILD_REPO_NAME`,
+                  `cd $(Build.Repository.Name)/$FAB_SAFE_SERVICE_NAME`,
```

Comment:
Is this working if I have 2 rings release at the same time?
What is the reason for not set image tag in ring directory ?

Reply:
I agree. We should probably set the config down at: `cd $APPLICATION_REPO_NAME/$FAB_SAFE_SERVICE_NAME/$RING_NAME` 

with:
`export APPLICATION_REPO_NAME=$(echo $(Build.Repository.Name) | tr '[:upper:]' '[:lower:]' | tr . - | tr / -)`,
and
`export RING_NAME=$(echo $(Build.SourceBranchName) | tr '[:upper:]' '[:lower:]' | tr . - | tr / -)`,

Though we need to double check what naming rules we have in place for rings. I think underscore's break it, so we may need to cover that case too.

@bnookala @evanlouie

Reply:
I raised this question with @bnookala, and wasn't sure if it needed to be at the ring level. Happy to make the changes if needed! :)

Reply:
underscores do break - the name has to be dns compliant.

as for which level, let's do ring level as @luongbac301092 had suggested

Reply:
Sounds good! I'll make that change.

Reply:
some reference on the ring name needing to be massaged: https://github.com/microsoft/bedrock/issues/1103